app/views/admin/badges/edit.html.erb

```diff
@@ -6,34 +6,32 @@
   </div>
 </header>
 
-<div class="crayons-card p-6">
+<div class="crayons-card crayons-card--content-rows">
   <%= form_for [:admin, @badge], method: :patch do |form| %>
-    <div class="form-group">
-      <%= form.label :title %>
-      <%= form.text_field :title, class: "form-control" %>
+    <div class="crayons-field mb-3">
+      <%= form.label :title, "Title:", class: "crayons-field__label" %>
+      <p class="crayons-field__description">
+        Badge's link is auto-generated based on the title.
```

Comment:
Non-blocking suggestion: should we include the link here? Do admins tend to want to _know_ or end up looking for the badge's link? If not, then it's probably more noise than it's worth, but if they do tend to look for that link, it might be worth including!

Somewhat unrelatedly...do we tend to use "link" or "url" to refer to these kinds of things? That's a small thing, but I see both versions of it used in `/app/views/admin/*`, so we might want to pick one or the other at some point for consistency's sake :)

Reply:
> should we include the link here?
It's a good idea! I'll add it.

> do we tend to use "link" or "url"
I'm not sure what we have consistently to be honest... I'll take a peek. Generally leaning toward "link" though since I think most people know what a link is, and not every Forem creator will know what a URL is.